### PR TITLE
More Psion shit and Cassad nerf

### PR DESCRIPTION
**Pull request by Kazkin**
-Cassad, after some examination, was found to be stupidly powerful. It's number of shots per battery has been reduced by 50%
-Psi tool/weapon creation is now split into melee, tool, and weapon with 6 unique options for all 3.
-Updated the Fallen Empress to have a slightly modified sprite.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Kazkin** [comment]
I coded this half asleep at night, can someone check this for errors?

**Comment by r4d6** [comment]
> I coded this half asleep at night, can someone check this for errors?

Sure thing

File: code/modules/psionics/psionic_items.dm [hunk 1]
```diff
@@ -249,6 +284,37 @@
 		list(mode_name="armor penetrating", projectile_type=/obj/item/projectile/plasma, fire_sound='sound/weapons/Laser.ogg', fire_delay=12, icon="kill", projectile_color = "#461266"),
 	)
 
+/obj/item/gun/energy/gun/martin/cult
+	name = "\"Star\" psi-laser pistol"
+	desc = "A laser weapon formed from the mind of a psion who shaped an oddity into a weapon. A psionic copy of a martin, an oft overlooked and ignored weapon. Perhaps this is formed from the \
```
Comment (r4d6) [inline code/modules/psionics/psionic_items.dm:289]:
```suggestion
	desc = "A laser weapon formed from the mind of a psion who shaped an oddity into a weapon. A psionic copy of a martin, an often overlooked and ignored weapon. Perhaps this is formed from the \
```

File: code/modules/psionics/standardpowers.dm [hunk 2]
```diff
@@ -152,7 +152,7 @@
 /obj/item/organ/internal/psionic_tumor/proc/psionic_weapon()
 	set category = "Psionic powers"
 	set name = "Psionic Weapon Construction (3)"
```
Comment (r4d6) [inline code/modules/psionics/standardpowers.dm:154]:
```suggestion
	set name = "Psionic Range Construction (3)"
```

File: code/modules/psionics/psionic_items.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/modules/psionics/psionic_items.dm:289]: "Oft" isn't a grammar mistake tho, just an archaic way of saying "often"

Comment (r4d6) [reply code/modules/psionics/psionic_items.dm:289]: I still think that 'often' sound better.

Comment (Kazkin) [reply code/modules/psionics/psionic_items.dm:289]: Gonna leave it as oft since its psionics,

File: code/modules/psionics/standardpowers.dm (hunk 2 above)
Comment (Kazkin) [reply code/modules/psionics/standardpowers.dm:154]: The name was too long when called "Psionic range weapon construction" and range in place of weapon is a little harder to understand versus melee and tool for the other powers.

Comment (r4d6) [reply code/modules/psionics/standardpowers.dm:154]: Fair enough

**Pull request merged**